src/stories/components/Branding/styles.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Branding
+
+.branding {
+  font-size: $font-size-xl;
+  font-weight: bold;
+  font-family: $header-font-family;
+}
+
+.logo {
```

Comment:
In terms of the alignment of the logo, right now it is not perfectly aligned vertically inside the anchor element. In other words, this makes the logo appear higher compared to how it shows on the Sketch design. I wonder if we want the logo to be perfectly aligned vertically. If this is the case I might need help from someone who is experienced in little styling details like this one.

Reply:
I tried to add some styling to the .logo and it fixes your issue. You can try this:
.logo {
   margin-right: rem(5px);
   position: absolute;
   top: rem(3px);
}

Reply:
@fernandowinfield you can do the following:
```
.branding {
  align-items: center;
  display: inline-flex;
}
```